client/src/components/HistoryViewer/HistoryViewer.scss

```diff
@@ -48,8 +52,13 @@
   white-space: nowrap;
 }
 
-// Bring any CMS tabs in the detail view closer to the table above it
 .history-viewer__version-detail {
+  &.history-viewer__version-detail--compare {
+    margin-left: $grid-gutter-width / 2;
+    margin-right: $form-check-input-gutter;
```

Comment:
Can you use `$grid-gutter-width` too here please? =)

If you need to divide it then you could use `$panel-padding-x` which has done this already